README.md

```diff
@@ -1,20 +1,23 @@
 [![REUSE status](https://api.reuse.software/badge/github.com/cap-java/sdm)](https://api.reuse.software/info/github.com/cap-java/sdm)
 
 # CAP plugin for SAP Document Management Service
-The project is currently in the development stage, where our team is actively working on building and refining its core components to ensure it meets all specified requirements and objectives.
-
 The **@cap-java/sdm** package is [cds-plugin](https://cap.cloud.sap/docs/java/cds-plugins#cds-plugin-packages) that provides an easy CAP-level integration with [SAP Document Management Service](https://discovery-center.cloud.sap/serviceCatalog/document-management-service-integration-option). This package supports handling of attachments(documents) by using an aspect Attachments in SAP Document Management Service.  
 This plugin can be consumed by the CAP application deployed on BTP to store their documents in the form of attachments in Document Management Repository.
 
 ## Key features
 
 - Create attachment : Provides the capability to upload new attachments.
+- Read attachment : Provides the capability to preview attachments.
+- Delete attachment : Provides the capability to remove attachments.
+- Rename attachment : Provides the capability to rename attachments.
 - Virus scanning : Provides the capability to support virus scan for virus scan enabled repositories.
 - Draft functionality : Provides the capability of working with draft attachments.
+- Display attachments specific to repository: Lists attachments contained in the repository that is configured with the CAP application.
 
 ## Table of Contents
 
 - [Pre-Requisites](#pre-requisites)
```

Comment:
In the pre-requisites, is the following needed?
"SAP Build WorkZone should be subscribed to view the HTML5Applications."

Reply:
Also remove [!Note] keyword.